scripts/tests/run_python_test.py

```diff
@@ -87,7 +88,8 @@ def DumpProgramOutputToQueue(thread_list: typing.List[threading.Thread], tag: st
               help='Script arguments, can use placeholders like {SCRIPT_BASE_NAME}.')
 @click.option("--script-gdb", is_flag=True,
               help='Run script through gdb')
-def main(app: str, factoryreset: bool, factoryreset_app_only: bool, app_args: str, script: str, script_args: str, script_gdb: bool):
+@click.option("--ci", is_flag=True, help="Use this flag in CI to suppress output from passing tests")
```

Comment:
let's use some different name then "CI". If we want to suppress output from passing tests, how about "silent" ?

Reply:
changed to "quiet" because it's not strictly silent. Updated the msg to match.